proto/buf.yaml

```diff
@@ -14,6 +20,9 @@ lint:
     - RPC_RESPONSE_STANDARD_NAME
     - RPC_REQUEST_RESPONSE_UNIQUE
     - COMMENT_MESSAGE
+  # TODO: Why do we ignore tendermint here?
+  ignore:
+    - tendermint
```

Comment:
this was to prevent linter errors, but since we're deleting the files we can remove it here too